blog-posts/2020-04-30-Post-SwaggerUI.md

```diff
@@ -3,7 +3,7 @@ title: "Swagger UI"
 author: "Kusuma Ghosh Dastidar"
 date: "2020-04-30"
 ---
-Swagger UI is the interface to visualize and interact with the application's REST APIs. The appearence of the Swagger UI that is generated from the swagger contract is provided by [Swagger](https://swagger.io/tools/swagger-ui/). Micro Focus projects would benefit from having the company specific branding and uniform appearence of their Swagger interfaces.
+Swagger UI is the interface to visualize and interact with a service’s REST APIs. The appearance of the Swagger UI generated from the Swagger contract is provided by [Swagger](https://swagger.io/tools/swagger-ui/). You can update the UI to add company branding for continuity across your Micro Focus Swagger interfaces.
 
 ### Swagger UI
-This project is a Micro Focus branded version of the Swagger UI. It is available for consumption as a WebJar or an NPM package and can be easily integrated in Spring Boot, Dropwizard or Tomcat applications. It has the necessary hooks to override swagger configuration params to allow applications to further customize the appearence of the Swagger UI. With this module it is easy to get updates from both Swagger and the Micro Focus styling.
+This project is a Micro Focus branded version of the Swagger UI. It is available as a WebJar or an NPM package and can be easily integrated in Spring Boot, Dropwizard or Tomcat applicationsand services. Using the included hooks to override Swagger configuration parameters, you can further customize the appearance of the Swagger UI. You can also easily implement updates to both the Swagger and Micro Focus styling.
```

Comment:
Add a comma after Dropwizard (we use the Oxford comma rule).
Need space between "applications" and "and" in "or Tomcat applicationsand services."